PWGCF/FemtoDream/FemtoDreamContainer.h

```diff
@@ -129,13 +147,44 @@ class FemtoDreamContainer
     }
   }
 
+  template <typename T>
+  void initMC(HistogramRegistry* registry, T& kstarBins)
+  {
+    mHistogramRegistry = registry;
+    std::string femtoObs;
+    if constexpr (mFemtoObs == femtoDreamContainer::Observable::kstar) {
+      femtoObs = "#it{k*} (GeV/#it{c})";
+    }
+    framework::AxisSpec femtoObsAxis = {kstarBins, femtoObs.c_str()};
+    std::string folderName = static_cast<std::string>(mFolderSuffix[mEventType]);
+    mHistogramRegistry->add((folderName + "kstar_resolution").c_str(), "; #it{k} _{T} reconstructed (GeV/#it{c}); #it{k} _{T} truth (GeV/#it{c})", kTH2F, {femtoObsAxis, femtoObsAxis});
```

Comment:
Shouldn't these lines also be part of the `if`?
I guess at some point you also expect other observables, right?